packages/ui/src/theme/tokens/index.ts

```diff
@@ -59,7 +62,7 @@ export interface Tokens {
  * The fully setup theme tokens. It has the same shape as Tokens
  * but each token has added fields.
  */
-export interface WebTokens extends Tokens {
+export interface WebTokens {
```

Comment:
Decoupling `WebTokens` from the `Tokens` interface allows for properties of `WebTokens` to only expect `WebDesignToken`, and `Tokens` to utilize `DesignToken`/`string` without getting TS errors on this line due to  `WebTokens` not extending `Tokens` correctly